client/server/pages/index.js

```diff
@@ -476,65 +476,84 @@ function setUpLoggedInRoute( req, res, next ) {
 }
 
 /**
- * Sets up a Content Security Policy header
+ * Sets up a Content Security Policy header for all Calypso routes
+ *
+ * This CSP is currently in REPORT-ONLY mode, which means violations are logged but not blocked.
+ * This allows us to identify issues before enforcing the policy.
+ *
+ * Required for compliance on pages handling credit card information.
  * @see https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP
  * @param {Object} req Express request object
  * @param {Object} res Express response object
  * @param {Function} next a callback to call when done
  */
 function setUpCSP( req, res, next ) {
-	const originalUrlPathname = req.originalUrl.split( '?' )[ 0 ];
-
-	// We only setup CSP for /log-in* for now
-	if ( ! /^\/log-in/.test( originalUrlPathname ) ) {
-		next();
-		return;
-	}
-
-	// This is calculated by taking the contents of the script text from between the tags,
-	// and calculating SHA256 hash on it, encoded in base64, example:
-	// `sha256-${ base64( sha256( 'window.AppBoot();' ) ) }` === sha256-3yiQswl88knA3EhjrG5tj5gmV6EUdLYFvn2dygc0xUQ
-	// you can also just run it in Chrome, chrome will give you the hash of the violating scripts
-	const inlineScripts = [ 'sha256-ZKTuGaoyrLu2lwYpcyzib+xE4/2mCN8PKv31uXS3Eg4=' ];
```

Comment:
Which script was this? Just double-checking that it's one of the ones we're now applying a `nonce` attribute to.

For what it's worth, I think that's a much better approach to use the inline `nonce` attribute than to try to manage all the hashes here.